src/screens/Library/components/InstallModal/index.tsx

```diff
@@ -382,12 +382,14 @@ export default function InstallModal({
         )
         if (Array.isArray(newWineList)) {
           setWineVersionList(newWineList)
-          if (
-            !newWineList.some(
-              (newWine) => wineVersion && newWine.bin === wineVersion.bin
-            )
-          ) {
-            setWineVersion(undefined)
+          if (wineVersion?.bin) {
```

Comment:
only do this check if we actually have a wineVersion, it was creating some issues with race conditions and sometimes the wineVersion setting got lost showing the wrong wineVersion preselected